### ISSUE TITLE
News Release Detail page accessibility improvements

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17185]
## Description
A question was brought to my attention regarding the News release detail page. When taking a look I saw that there are some improvements that would greatly improve the accessibility of the template

- "Media Contacts" and "Download media assets" are currently just bold text. They are however different sections of the page and should be H2 headings.
- Within the Download assets section, we recently updated these links to use the `<va-link>` component with the `download` and `filetype` props.
  - However, we want the `filetype` to dynamically pull the prop type. The only way to do this right now would be to parse this from the name of the file which isn't super reliable and might also need to be maintained often if the allowed filetypes in Drupal changed.
  - We should look into doing this in a more reliable way - this wasn't in scope in the previous ticket - which is why we are bringing it up here.
- If the editor adds an image to be downloaded, it would also be very good to be able to display that image somewhere. This would allow folks using assistive tech to be able to access the alt text. It could be a thumbnail within the download area, it could be a larger image somewhere in the main article, it could be something else entirely.

Example:
- [Back end](https://prod.cms.va.gov/boston-health-care/news-releases/clinician-attitudes-towards-telehealth-may-impact-how-patients-receive-care)
- [Front end](https://www.va.gov/boston-health-care/news-releases/clinician-attitudes-towards-telehealth-may-impact-how-patients-receive-care/)

### Other considerations
- Any front end changes, we would want to make sure are reflected in the View screen on Drupal so this may require a back end ticket as well
- It may be a good idea to have Design mock something up, especially since adding the image isn't as straightforward as the other updates.

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [x] Facilities
- [ ] Accelerated Publishing
